src/components/NodeDetails/NodeDetails.tsx

```diff
@@ -294,11 +294,13 @@ function getCommand(
       if (
         instanceParameters[i].data.param_name === pluginParameters[j].data.name
       ) {
-        const boolean = instanceParameters[i].data.type === "boolean";
+        const isBoolean = instanceParameters[i].data.type === "boolean";
+        const isString = instanceParameters[i].data.type === "string";
+        const value = instanceParameters[i].data.value;
 
         modifiedParams.push({
           name: pluginParameters[j].data.flag,
-          value: boolean ? "" : instanceParameters[i].data.value,
+          value: isBoolean ? " " : isString ? `'${value}'` : value,
```

Comment:
Adding single quotes won't always work, e.g. what if `value` itself contains a single quote character?

I recommend using a library for this, e.g. https://www.npmjs.com/package/shlex